service/src/main/java/greencity/controller/NotificationController.java

```diff
@@ -0,0 +1,69 @@
+package greencity.controller;
+
+
+import greencity.dto.notification.NotificationDto;
+import greencity.entity.Notification;
+import greencity.service.NotificationService;
+import greencity.service.UserService;
+import jakarta.validation.constraints.NotEmpty;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.AccessDeniedException;
+import org.springframework.web.bind.annotation.*;
+
+import java.security.Principal;
+import java.util.List;
+
+@RestController
+@RequestMapping("/notifications")
+@RequiredArgsConstructor
+public class NotificationController {
+    private final NotificationService notificationService;
+    private final UserService userService;
+
+    /** Отримати всі нотифікації поточного користувача. */
```

Comment:
comments are usually in english
```suggestion
    /** Retrieve all notifications for the current user. */
```